src/App.tsx

```diff
@@ -69,6 +86,11 @@ const App = () => {
 		typeof Descope
 	>['theme'];
 
+	const faviconUrl = process.env.REACT_APP_FAVICON_URL || '';
```

Comment:
I think it can be good to make the default `https://static.descope.com/pages/{projectId}/v2-beta/{ssoAppId}/assets/favicon.ico` so it will also work for someone that clone the repo (also in the PR of the other branch)

Reply:
+1